src/main/java/cofh/thermaldynamics/duct/fluid/FluidGrid.java

```diff
@@ -245,7 +245,7 @@ public FluidStack getNodeShare(DuctUnitFluid theCond) {
 
 	public int getNodeAmount(DuctUnitFluid theCond) {
 
-		if (theCond.canStoreFluid()) {
+		if (!theCond.canStoreFluid()) {
```

Comment:
context: inverted the logic since this would cause fluidducts to no save the fluid amount on writeNbt